client/my-sites/checkout/composite-checkout/components/prepurchase-notices/index.tsx

```diff
@@ -129,8 +158,8 @@ const PrePurchaseNotices = () => {
 	return null;
 };
 
-const Wrapper = ( props ) => {
-	const notice = PrePurchaseNotices( props );
+const Wrapper = () => {
+	const notice = PrePurchaseNotices();
```

Comment:
I know this already existed and it's not wrong, but it's a really unusual way to render a React component... it might be nice in a follow-up to refactor this so that there's a separate hook that determines if `PrePurchaseNotices` should be rendered, then move this to `<PrePurchaseNotices />` down inside the JSX. Just figured I'd mention it. Glad this is TS now!

Reply:
Yes agreed, I will look into refactoring this in a followup PR.  Thanks sirbrillig! 👍